graft.cabal

```diff
@@ -14,31 +14,33 @@ build-type:          Simple
 cabal-version:       >=1.10
 
 library
+  default-extensions:  FlexibleInstances GADTs TemplateHaskell TypeSynonymInstances ExistentialQuantification OverloadedStrings
   hs-source-dirs:      src
   exposed-modules:     Graft.Types, Graft
-  build-depends:       base >= 4.7 && < 5
-                     , mtl >= 2.2.1
-                     , text >= 1.2.2.1
-                     , containers >= 0.5.6.2
-                     , lens >= 4.13
-                     , directory >= 1.2.2.0
-                     , parsec >= 3.1.11
+  build-depends:       base
+                     , mtl
+                     , text
+                     , containers
+                     , lens
+                     , directory
+                     , parsec
   default-language:    Haskell2010
 
 test-suite graft-test
+  default-extensions:  FlexibleInstances GADTs TemplateHaskell TypeSynonymInstances ExistentialQuantification OverloadedStrings
```

Comment:
Nit: Could we list these in alphabetical order so it’s easier to scan (binary search 😉)?

Reply:
Done.